xbmc/cores/AudioEngine/DSPAddons/ActiveAEDSP.cpp

```diff
@@ -93,16 +92,19 @@ void CActiveAEDSP::OnApplicationMessage(KODI::MESSAGING::ThreadMessage* pMsg)
     case TMSG_SETAUDIODSPSTATE:
       if(pMsg->param1 == ACTIVE_AE_DSP_STATE_ON)
       {
-        if(pMsg->param1 == ACTIVE_AE_DSP_ASYNC_ACTIVATE)
+        if (!IsActivated()) //< Prevent if already in activated state
         {
-          Activate(true);
-        }
-        else
-        {
-          Activate();
+          if(pMsg->param1 == ACTIVE_AE_DSP_ASYNC_ACTIVATE)
+          {
+            Activate(true);
+          }
+          else
+          {
+            Activate();
+          }
         }
       }
-      else if(pMsg->param1 == ACTIVE_AE_DSP_STATE_OFF)
+      else if(pMsg->param1 == ACTIVE_AE_DSP_STATE_OFF && IsActivated()) //< Prevent if already in deactive state with IsActivated check
```

Comment:
Yes is better on Activate, I'm was a bit fast and has only put to the source of the faults.